megamek/src/megamek/client/ratgenerator/MissionRole.java

```diff
@@ -627,7 +627,6 @@ public static MissionRole parseRole(String role) {
             case "raider":
                 return RAIDER;
             case "incendiary":
-            case "incindiary":
```

Comment:
Let's keep this in case there's some misspelled data somewhere.

Reply:
Sounds good :)